plugins/inputs/couchdb/couchdb.go

```diff
@@ -80,7 +80,9 @@ type (
 	}
 
 	CouchDB struct {
-		Hosts []string `toml:"hosts"`
+		Hosts         []string `toml:"hosts"`
+		BasicUsername string   `toml:"basic_username"`
```

Comment:
I probably would have just called it Username/Password to stay uniform with other plugins (elasticsearch, kibana, icinga2, rabbitmq, etc...)

Reply:
I was on the fence about this but decided to be more explicit because CouchDB supports several [authentication methods](http://docs.couchdb.org/en/2.2.0/api/server/authn.html#).  We have a number plugins using this naming as well and I could go either way.

I would probably never consider implementing the cookie method, but the proxy method is a possibility.  I guess if we added it we could provide another option `auth_method`?  Which way do you think is best?

Reply:
Sound logic. I say leave it, as you said, there are other plugins that use the `basic_` prefix.